cds_sorenson/utils.py

```diff
@@ -101,8 +103,9 @@ def get_status(job_id):
 
 def _get_preset_config(preset_id):
     """Return preset config based on the preset_id."""
-    PRESETS = current_app.config['CDS_SORENSON_PRESETS']
-    presets_list = [item for presets in PRESETS.values() for item in presets]
-    for preset in presets_list:
-        if preset_id == preset.get('preset_id'):
-            return preset
+    return [
```

Comment:
Can you, please, split that into parts that can be understand.
```
presets = current_app.config['CDS_SORENSON_PRESETS'].values()
configs = [preset.values() for preset in presets],
etc.
```
Unless this style of programming is a convention that we are following in Invenio (@egabancho ?)?